class.jetpack-xmlrpc-server.php

```diff
@@ -70,9 +70,31 @@ function xmlrpc_methods( $core_methods ) {
 	function bootstrap_xmlrpc_methods() {
 		return array(
 			'jetpack.verifyRegistration' => array( $this, 'verify_registration' ),
+			'jetpack.remoteAuthorize' => array( $this, 'remote_authorize' ),
 		);
 	}
 
+	function remote_authorize( $request ) {
```

Comment:
This function won't work as written (except it does because of [this bug](https://github.com/Automattic/jetpack/pull/3364/files#r52968027).  We need to set the current user before we attempt to verify the nonce.